src/main/scala/beam/agentsim/agents/ridehail/AsyncGreedyVehicleCentricMatching.scala

```diff
@@ -84,10 +86,16 @@ class AsyncGreedyVehicleCentricMatching(
               combinations.append(t.matchId)
               val cost = computeCost(t)
               if (tripsWithKPassengers.size == solutionSizePerPool) {
-                // then replace the trip with highest sum of delays
```

Comment:
We should keep the comment or turn it into a method named aptly

Reply:
It just changed the line number. Some github issue makes it keep showing
![image](https://user-images.githubusercontent.com/12264098/93909221-ec3ee580-fcd5-11ea-941d-8169dc1c7217.png)

Reply:
sorry about that :)